src/main/java/uk/gov/hmcts/reform/et/syaapi/notification/NotificationsProperties.java

```diff
@@ -65,12 +65,17 @@ public class NotificationsProperties {
     @Value("${notifications.tribunalResponse}")
     @NotBlank
     private String tribunalResponseTemplateId;
-    
+
     @Value("${notifications.claimantResponseNo}")
     @NotBlank
     private String claimantResponseNoTemplateId;
 
     @Value("${notifications.claimantResponseYes}")
     @NotBlank
     private String claimantResponseYesTemplateId;
+
+    @Value("${notifications.respondentResponse}")
```

Comment:
I believe you will need to add a PMD Suppression for this file
`@SuppressWarnings("PMD.TooManyFields")`